### PR TITLE
Update dependabot.yml to Add Swift Package Manager

### PR DESCRIPTION
**Pull request by gyasistory**
https://github.com/dependabot/dependabot-core/issues/1245#issuecomment-1650556093

https://github.com/dependabot/dependabot-core/pull/7525

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -10,3 +10,13 @@ updates:
   assignees:
   - frett
   - levieggertcru
+- package-ecosystem: swift
```
Comment (frett) [inline .github/dependabot.yml:13]: The ticket you linked mentioned that the swift support is only for Package.swift and not XCode.

That's similar support that renovate has which doesn't currently cover apps using spm

Comment (frett) [reply .github/dependabot.yml:13]: relevant dependabot ticket: https://github.com/dependabot/dependabot-core/issues/7694

**Review by frett — APPROVED** [review]
I'll approve since I don't think this will hurt anything, but I don't think this SPM support will actually do anything for the godtools-swift repo

File: .github/dependabot.yml (hunk 1 above)
Comment (gyasistory) [reply .github/dependabot.yml:13]: [Ok, I thought I found something.](https://github.com/dependabot/dependabot-core/pull/7525)

Comment (gyasistory) [reply .github/dependabot.yml:13]: @levieggertcru I will leave it up to you whether or not to merge it in.

**Pull request merged**